lib/cucumber/cli.js

```diff
@@ -89,6 +89,8 @@ function Cli(argv) {
 \n\
 --no-source                   Don't print the source uris.\n\
 \n\
+--[no-]colors                 Enable/disable colors in output.\n\
```

Comment:
I don't think we need to copy cucumber-ruby to a T. We already specify formats differently. I also think no-colors is better, so why not as cucumber-ruby to rename? cucumber-jvm uses `--monochrome` as well. 

I like the idea of keeping these implementations interfaces the same, but these are also built at different times and I'd prefer not to be locked to the cucumber-ruby interface.